### PR TITLE
Removed fastlane code/examples

### PR DESCRIPTION
**Pull request by sandraleon**
Also refactored a bit.

### CODE REVIEW

File: ads/rubicon.js [hunk 1]
```diff
@@ -27,90 +24,16 @@ import {loadScript, validateData, writeScript} from '../3p/3p';
...
   // TODO: check mandatory fields
   validateData(data, [], [
-    'slot', 'targeting', 'categoryExclusions',
-    'tagForChildDirectedTreatment', 'cookieOptions',
-    'overrideWidth', 'overrideHeight', 'loadingStrategy',
-    'consentNotificationId', 'useSameDomainRenderingUntilDeprecated',
     'account', 'site', 'zone', 'size',
-    'pos', 'kw', 'visitor', 'inventory',
+    'kw', 'visitor', 'inventory',
     'type', 'method', 'callback',
```
Comment (harpere) [inline ads/rubicon.js]: Does `type` need to be validated?  I don't see it used anywhere.

File: ads/rubicon.md [hunk 2]
```diff
@@ -100,21 +71,6 @@ By default the ad size is based on the `width` and `height` attributes of the `a
 - `data-zone`
```
Comment (harpere) [inline ads/rubicon.md:71]: should we include `type` in this list of attributes?  I see it in the `ads-legacy.amp.html` and `ads.amp.html` examples, as well as `validator/testdata/feature_tests/ads.[html|out]`

**Review by harpere — APPROVED** [review]
A couple of very minor questions/comments, but overall looks good.

File: ads/rubicon.md (hunk 2 above)
Comment (sandraleon) [reply ads/rubicon.md:71]: See answer for other question.

File: ads/rubicon.js (hunk 1 above)
Comment (sandraleon) [reply ads/rubicon.js]: I think "type" here refers to the vendor the pub is using:
```
<amp-ad width=320 height=50
    type="rubicon"
    data-method="smartTag"
    data-account="14062"
    data-site="70608"
    data-zone="335918"
    data-size="43">
</amp-ad>

```
But good point, we don't need to validate that in our js file. Removing.

**Comment by sandraleon** [comment]
Will submit new PR to opensource repo instead of merging to master.

**Pull request closed without merging**